src/Features/Core/Portable/AddFileBanner/AbstractAddFileBannerCodeRefactoringProvider.cs

```diff
@@ -69,7 +69,8 @@ public override async Task ComputeRefactoringsAsync(CodeRefactoringContext conte
                 if (siblingBanner.Length > 0 && !siblingDocument.IsGeneratedCode(cancellationToken))
                 {
                     context.RegisterRefactoring(
-                        new MyCodeAction(c => AddBannerAsync(document, root, siblingDocument, siblingBanner, c)));
+                        new MyCodeAction(c => AddBannerAsync(document, root, siblingDocument, siblingBanner, c)),
+                        new Text.TextSpan(position, length: 0));
```

Comment:
shouldn't span be whole banner?

Reply:
It's to generate banner, and thus there's no banner in the file. And the applicableToSpan needs to correspond to original document not the document to be.